packages/repl/src/lib/workers/bundler/index.js

```diff
@@ -1,4 +1,5 @@
 import { rollup } from '@rollup/browser';
+import { exports as exprots_resolver, legacy as legacy_resolver } from 'resolve.exports';
```

Comment:
the typo is still there :laughing: 
```suggestion
import { exports as exports_resolver, legacy as legacy_resolver } from 'resolve.exports';
```